utilities/data/management_user.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "model": "management.user",
+    "pk": 2,
+    "fields": {
+      "password": "pbkdf2_sha256$870000$gumjpzt6qPeQoSNwKzoHR1$HCGrkpdrdzAI+1yK+hxZMYSKHrrqNiFtWon8DsZWdjc=",
```

Comment:
I'm sure it's fine, just seems a bit scary having a password in a repo...

Reply:
Sorted. I left a password since I thought that it would fail otherwise, but it seems to be working fine with an empty string...